src/main/java/com/cglx/Calculator.java

```diff
@@ -31,7 +31,14 @@ public int square(int a)
         return a * a;
     }
 
-    public boolean areEqual(float f1, float f2) {
-        return f1==f2;
+    private void readFile() {
+        FileInputStream fis = new FileInputStream(new File("TestFile.txt"));
+        
+    }
+
+    public int remainder(int a, int b)
+    {
+        int result = a%b;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AS1488)